infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/provisioner/installer/InstallerConfigApplier.java

```diff
@@ -83,11 +74,10 @@ public void apply(Environment envConfig, DockerEnvironment dockerEnvironment)
    * @param machine affected machine
    * @throws InstallerException if any error occurs
    */
-  public void apply(@Nullable MachineConfig machineConf, DockerContainerConfig machine)
+  public void apply(@Nullable InternalMachineConfig machineConf, DockerContainerConfig machine)
       throws InstallerException {
     if (machineConf != null) {
-      for (Installer installer :
-          installerRegistry.getOrderedInstallers(machineConf.getInstallers())) {
+      for (Installer installer : machineConf.getInstallers()) {
```

Comment:
Here is how we fetch installers: 
https://github.com/garagatyi/che/blob/2f83e9b0f5652fa6c759c149c47694f9960773e3/wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/spi/InternalMachineConfig.java#L72-L72